src/pages/home/report/comment/TextCommentFragment.tsx

```diff
@@ -66,54 +58,53 @@ function TextCommentFragment(props) {
 
         return (
             <RenderCommentHTML
-                source={props.source}
+                source={source}
                 html={htmlWithTag}
             />
         );
     }
 
     const containsOnlyEmojis = EmojiUtils.containsOnlyEmojis(text);
+    const message = isEmpty(iouMessage) ? text : iouMessage;
 
     return (
-        <Text style={[containsOnlyEmojis ? styles.onlyEmojisText : undefined, styles.ltr, ...props.style]}>
+        <Text style={[containsOnlyEmojis && styles.onlyEmojisText, styles.ltr, style]}>
             <ZeroWidthView
                 text={text}
-                displayAsGroup={props.displayAsGroup}
+                displayAsGroup={displayAsGroup}
             />
             <Text
                 style={[
                     containsOnlyEmojis ? styles.onlyEmojisText : undefined,
                     styles.ltr,
-                    ...props.style,
+                    style,
                     styleAsDeleted ? styles.offlineFeedback.deleted : undefined,
-                    !DeviceCapabilities.canUseTouchScreen() || !props.isSmallScreenWidth ? styles.userSelectText : styles.userSelectNone,
+                    !DeviceCapabilities.canUseTouchScreen() || !isSmallScreenWidth ? styles.userSelectText : styles.userSelectNone,
                 ]}
             >
-                {convertToLTR(props.iouMessage || text)}
+                {convertToLTR(message)}
             </Text>
-            {Boolean(fragment.isEdited) && (
+            {!!fragment.isEdited && (
```

Comment:
Interesting–what else could this be besides a Boolean? I wonder why we decided to do this originally.